README.md

```diff
@@ -31,6 +31,8 @@ This provider's versions are able to install and manage the following versions o
 |-|-|-|
 |OpenStack Provider v1alpha1 (ea309e7f)|✓|✓|
 
+Kubeadm and Kubelet versions are defined in the the file `cmd/clusterctl/examples/openstack/out/machines.yaml` which is generated by running the `generate-yaml.sh`.
```

Comment:
Can we highlight the version is in `spec.versions.kubelet`?

Reply:
Thanks, I updated.